src/_pytest/pastebin.py

```diff
@@ -65,7 +65,7 @@ def create_new_paste(contents):
     from urllib.request import urlopen
     from urllib.parse import urlencode
 
-    params = {"code": contents, "lexer": "python3", "expiry": "1week"}
+    params = {"code": contents, "lexer": "text", "expiry": "1week"}
```

Comment:
Hi @goerz, wondering, why is "python3" no longer recognized? I mean, is there another, or it doesn't really make any difference what the lexer is? 🤔

Reply:
“python3” is still recognized, and in fact it works fine for uploading pytest output most of the time. It just fails for specific outputs, like the one I included in the report for #5806. I haven’t really looked into the internals of [pinnwand](https://github.com/supakeen/pinnwand) (the backend for bpaste.net), but I would assume that the specified lexer is used to parse the pasted text (for syntax highlighting?). Obviously, the pytest output is never valid Python code, and I’m really not sure why this particular output causes bpaste to return an http error, while other, very similar output goes through without an error. In any case, though, the “text” lexer is definitely the semantically correct choice, as pytest output isn’t something that can be parsed.

Reply:
Fair enought, thanks!